docs/apis/http/tus_upload.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "tus upload"
+date: 2023-10-10T00:00:00+00:00
+weight: 21
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/apis/http
+geekdocFilePath: tus_upload.md
+geekdocCollapseSection: true
+---
+
+Infinite Scale supports [tus resumable-upload protocol](https://tus.io/protocols/resumable-upload).
```

Comment:
```suggestion
ownCloud Infinite Scale supports [tus resumable-upload protocol](https://tus.io/protocols/resumable-upload).
```

Reply:
ownCloud should not be part of the product name, simply use Infinite Scale.